extensions/common/api/api-core/src/main/java/org/eclipse/dataspaceconnector/api/query/QuerySpecDto.java

```diff
@@ -51,6 +59,7 @@ public Integer getLimit() {
         return limit;
     }
 
+    @Deprecated
```

Comment:
At this point I would deprecate the `QuerySpec.Builder.filter(String)` builder method too, to switch to the `QuerySpec.Builder.filter(String)` as well, as it contains some logic that doesn't worth maintaining anymore

Reply:
done